src/routes/+layout.server.ts

```diff
@@ -51,7 +51,7 @@ export const load: LayoutServerLoad = async ({ locals, depends, url }) => {
 			})
 			.map((conv) => ({
 				id: conv._id.toString(),
-				title: conv.title,
+				title: settings?.hideEmojiOnSidebar ? conv.title.replace(/\p{Emoji}/gu, "") : conv.title,
```

Comment:
It's a bit weird to hide there? I would hide at bottom level (Svelte component)

Reply:
Thinking about this more, I think it makes more sense to keep it at the top level.

Both the page title, top-level mobile nav, and `NavConversationItem` use this layout data as a source of truth, so I would prefer to change it once at the top level than in three places.